src/lib/y2partitioner/widgets/overview.rb

```diff
@@ -76,8 +122,11 @@ def disks_items
 
       def partition_items(disk)
         disk.partitions.map do |partition|
+          y2partition = Y2Storage::Partition.find_by_name(@device_graph, partition.name)
+          view = ExpertPartitioner::PartitionTreeView.new(y2partition)
+          pane_term = view.create
           id = "partition:" + partition.name
-          item_for(id, partition.sysfs_name)
+          item_for(id, partition.sysfs_name, pane: pane_term)
```

Comment:
to be honest. I really do not like this. This very break idea of CWM, you should pass here CWM object if you want display something.

Another part is that it is static and not dynamic, so if you modify something in staging you have to recreate all views, which is memory and also processing killer. My original idea with `partition:<name>` was to have in handle
```
when /^partition:/
  name = id.split(":",2)[1]
  object = PartitionView.new(name)
  replace_point.replace(object)
```

so it is created only when requested and not needed to be hold in memory and not constructed even when not needed